### PR TITLE
delete memento

### PR DESCRIPTION
**Pull request by hchoura**
no used
create an issue

### CODE REVIEW

**Comment by olhabn** [comment]
it is used when we press edit button to change memento from '....::view:id' to '.....::edit::id', isn't it?

File: src/foam/comics/v2/DAOSummaryView.js [hunk 1]
```diff
@@ -182,8 +182,6 @@ foam.CLASS({
       code: function() {
         if ( ! this.stack ) return;
 
-        if ( this.memento.tail )
```
Comment (kgrgreer) [inline src/foam/comics/v2/DAOSummaryView.js:185]: Why do you want to delete this?

**Comment by kgrgreer** [comment]
@olhabn Can you please review this?

**Comment by olhabn** [comment]
it is used when we press edit button to change memento from '....::view:id' to '.....::edit::id'.

please create a ticket if it creates a bug and I will implement "on edit memento" if a different way. thanks

File: src/foam/comics/v2/DAOSummaryView.js (hunk 1 above)
Comment (olhabn) [reply src/foam/comics/v2/DAOSummaryView.js:185]: it is used when we press edit button to change memento from '....::view:id' to '.....::edit::id'.

please create a ticket if it creates a bug and I will implement "on edit memento" if a different way. thanks

**Review by olhabn — APPROVED** [review]

**Pull request merged**